flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/TemporalTypesTest.scala

```diff
@@ -262,6 +263,30 @@ class TemporalTypesTest extends ExpressionTestBase {
       "2018-03-14")
   }
 
+  private def expectExceptionThrown(
+      sqlExpr: String,
+      expected: String,
+      keywords: String,
+      clazz: Class[_ <: Throwable] = classOf[ValidationException])
+  : Unit = {
```

Comment:
nit: align `: Unit` with `try` manually

Reply:
I noticed same style in our Flink code(eg `AggregateValidationTest`, `CorrelateValidationTest`), and if we align in this way, the last two lines of this function looks strange
```
 : Unit = {
 try {
    ...
 } catch {
    ....
 }
 }
```